src/pyenphase/models/system_production.py

```diff
@@ -58,3 +66,30 @@ def from_production(cls, data: dict[str, Any]) -> EnvoySystemProduction:
             ),
             watts_now=int(round(now_source["wNow"])),
         )
+
+    @classmethod
+    def from_production_phase(
+        cls, data: dict[str, Any], phase: int
+    ) -> EnvoySystemProduction | None:
+        """Initialize from the production API phase data.
+
+        :param data: JSON reply from /production endpoint
+        :param phase: Index (0-2) in [lines] segment for which to return data
+        :return: Lifetime, last seven days, todays energy and current power for production phase
+        """
+        all_production = data["production"]
+        eim = find_dict_by_key(all_production, "eim")
+
+        # if {production[type=eim]{Lines[]} or phase is missing return None
+        phases = eim.get("lines")
+        if not phases or phase >= len(phases):
+            return None
+
+        return cls(
+            watt_hours_lifetime=int(round(phases[phase].get("whLifetime") or 0)),
+            watt_hours_last_7_days=int(
+                round(phases[phase].get("whLastSevenDays") or 0)
+            ),
+            watt_hours_today=int(round(phases[phase].get("whToday") or 0)),
+            watts_now=int(round(phases[phase].get("wNow") or 0)),
```

Comment:
```suggestion
            watt_hours_lifetime=int(round(phase_data.get("whLifetime") or 0)),
            watt_hours_last_7_days=int(
                round(phase_data.get("whLastSevenDays") or 0)
            ),
            watt_hours_today=int(round(phase_data.get("whToday") or 0)),
            watts_now=int(round(phase_data.get("wNow") or 0)),
```

black may complain about this suggestion